CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing
```

Comment:
I think there has been some messup in terms of making this PR? There are changes to the readme and all which is not relevant to this PR.

Reply:
Its a subbranch of the README PR, we can merge this after that one